### PR TITLE
Update projects.toml

### PR DESCRIPTION
**Pull request by Caveats**
Added Polaron World Engine

### CODE REVIEW

**Comment by Caveats** [comment]
Added Polaron World Engine

**Comment by Caveats** [comment]
Added Polaron Engine.

File: content/wiki/projects.toml [hunk 1]
```diff
@@ -253,6 +253,15 @@ state = "Beta"
...
 
+[[projects]]
+name = "Polaron World Engine"
+url = "https://polaronworldengine.com/"
+state = "Active Dev."
+source = "https://gitlab.com/veloren/veloren"
+license = "Commercial"
+language = ["C++"]
+rendering = "Ray Marching, Ray Casting"
+
```
Comment (Longor1996) [inline content/wiki/projects.toml:264]: New projects are supposed to go at the bottom of their respective status section (Active Development in this case).

File: content/wiki/projects.toml [hunk 2]
```diff
@@ -253,6 +253,15 @@ state = "Beta"
 ################### Active Development ####################
 ###########################################################
 
+[[projects]]
+name = "Polaron World Engine"
+url = "https://polaronworldengine.com/"
+state = "Active Dev."
+source = "https://gitlab.com/veloren/veloren"
```
Comment (ShinySilver) [inline content/wiki/projects.toml:260]: This address doesn't seem to be right. Is veloren using polaron world engine?

Comment (Caveats) [reply content/wiki/projects.toml:260]: Apologies - yes that's a typo!

**Pull request closed without merging**